src/audio/up_down_mixer/up_down_mixer.c

```diff
@@ -25,6 +25,13 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+#endif
+#include <sof/audio/module_adapter/module/generic.h>
+#include <module/module/base.h>
+#include <module/module/api_ver.h>
+#include <rimage/sof/user/manifest.h>
+#include <module/audio/source_api.h>
+#include <module/audio/sink_api.h>
```

Comment:
these last 5 headers shouldn't be needed in a monolithic build, right? Should there be an `#else`?

Reply:
Sorry, somehow I've mixed two PRs.